includes/compat/amp.php

```diff
@@ -38,7 +38,7 @@
 	 * @return bool Whether it is singular story post (and thus an AMP endpoint).
 	 */
 	function is_amp_endpoint(): bool {
-		return is_singular( Story_Post_Type::POST_TYPE_SLUG );
+		return is_singular( Story_Post_Type::POST_TYPE_SLUG ) && ! is_embed() && ! post_password_required();
```

Comment:
Should we do this now?
```suggestion
                 $context = new Context();
		return $context->is_web_story();
```

Reply:
Context can't be used here. As this function is loaded extremely early in the bootstrap.

Reply:
I can add an inline comment for it